controller/QtiCreator.php

```diff
@@ -348,4 +363,66 @@ private function validateXmlInput(string $xml)
         }
         \tao_helpers_Xml::getSimpleXml($xml);
     }
+
+    /**
+     * Download sample template for tabular import
+     *
+     * @param string
+     * @throws common_exception_Error
+     */
+    public function downloadCsv()
```

Comment:
```suggestion
    /**
     * @throws common_exception_Error
     */
    public function downloadCsv(): string
```

We prefer declarative type hint and return type over annotations always.

But, another topic. Do not you think we should have a new controller for this? This controller is about QTICreator and what we want is to download a sample for CSVImport.

Wdyt?

Reply:
Suggested change completed